### ISSUE TITLE
Can you share Multi-Scale Grouping Classification config file ?

### ISSUE THREAD

**Issue by Number1JT** [Yochengliu/Relation-Shape-CNN#40]
We have reproduce the reported performance of single scale group for classification task using the MXNet framework, however we tried many times but still can not reproduce the performance under multi-scale config . Can you share your  config file ? (such as radius, mlp layers ..etc). Any suggestions are appreciated.

**Comment by Yochengliu** [comment]
Hi, many thanks for your effort. It sounds exciting of your work.
The performance of multi-scale version has uncontrollable randomness, due to some uncontrollable functions in cuda kernal. Thus this version is difficult to meet the bar of stability in our view of code releasing. We had been trying to fix the randomness problem, but nothing works. Hope you professional guys do it :)

class RSCNN_MSN(nn.Module):
    def __init__(self, num_classes, input_channels=0, relation_prior=1, use_xyz=True):
        super().__init__()

        self.SA_modules = nn.ModuleList()
        
        self.SA_modules.append(
            PointnetSAModuleMSG(
                npoint=512,
                radii=[0.175, 0.26, 0.4],
                nsamples=[32, 64, 128],
                mlps=[[input_channels, 128], [input_channels, 128], [input_channels, 128]],
                first_layer=True,
                use_xyz=use_xyz,
                relation_prior=relation_prior
            )
        )

        input_channels = 128 * 3
        self.SA_modules.append(
            PointnetSAModuleMSG(
                npoint=128,
                radii=[0.21, 0.33, 0.5],
                nsamples=[32, 64, 128],
                mlps=[[input_channels, 256], [input_channels, 256],
                      [input_channels, 256]],
                use_xyz=use_xyz,
                relation_prior=relation_prior
            )
        )
        
        input_channels = 256 * 3
        self.SA_modules.append(
            # global convolutional pooling
            PointnetSAModule(
                nsample = 128,
                mlp=[input_channels, 1024], 
                use_xyz=use_xyz
            )
        )

        self.FC_layer = nn.Sequential(
            nn.Dropout(p=0.5),
            pt_utils.FC(1024, 512, activation=nn.ReLU(inplace=True), bn=True),
            nn.Dropout(p=0.5),
            pt_utils.FC(512, 256, activation=nn.ReLU(inplace=True), bn=True),
            nn.Dropout(p=0.5),
            pt_utils.FC(256, num_classes, activation=None)
        )

    def _break_up_pc(self, pc):
        xyz = pc[..., 0:3].contiguous()
        features = (
            pc[..., 3:].transpose(1, 2).contiguous()
            if pc.size(-1) > 3 else None
        )
        return xyz, features

    def forward(self, pointcloud: torch.cuda.FloatTensor):
        xyz, features = self._break_up_pc(pointcloud)
        for module in self.SA_modules:
            xyz, features = module(xyz, features)
        return self.FC_layer(features.squeeze(-1))
